SmartPlaces.Facilities/product/Topology/product/src/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="Program.cs" company="Microsoft">
+//   Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Contoso.Topology
```

Comment:
We should probably be consistent across namespaces. What we were working on made sense, open sourcing everything.. not so much.